Resume.php

```diff
@@ -0,0 +1,88 @@
+            resources that Washington University’s Health and Wellness team have to offer, they do not consider these programs
+            mental health resources. Most students consider SHS counseling and Uncle Joe’s to be the only mental health resources
+            on campus suggesting a lack in knowledge about the full extent of all programs that are offered and a barrier in accessing
+            these resources. In addition there seems to be a stigma that is associated with mental health on campus. The stigma mostly
+            seems to revolve around the high standard that students at Washington University are held to and the need to be “perfect”. I
+            recommend more aggressively targeting informational resources at incoming freshman; bringing mental health into the classroom;
+            eliminating the barriers that stand in students’ way of taking advantage of these resources; and finally
+            marketing toward the everyday life problem, not just sever psychological conditions. I believe these recommendations
+            will help eliminate the stigma associated with being perfect and increase the accessibility of mental health resources on campus.
+    </p>
+    <h4 class="projects">
+        Coding Fellowship: LessAnnoyingCRM (2017):
+</h4>
+        <p class="description">
+            I spent the summer leraning how to code and created this blog
+        </p>
+    <h4 class="projects">
+        Happy and Neutral Mood Induction on Math and Recall Tests (2016):
+</h4>
+        <p class="description"> Does one’s temporary mood affect the way a person will perform on different types of tasks?
+            In our experiment we explored the effect of happy and neutral moods on 2 different task types, math problem
+            solving and memory recall. Sixteen Washington University students participated in the study all ranging in age
+             from 20-22.  We used a within-groups design, where each participant was shown a series of photos to induce
+             either a happy or neutral mood and then required to complete either a memory recall or math task. A Latin-square
+              design was used to make sure that every condition was shown in each place of the order. We found a main effect
+              of task type, p = .018. Participants performed significantly better on the math task as opposed to the memory
+              recall task.  No main effect of mood, p = .37, was seen. An induced happy or neutral mood did not appear to
+               cause a difference in performance. We found no interaction between mood and task type, p = .65. Overall our
+               findings show that mood does not affect one’s performance on different task types.
+<div class = "aboutMe">
+    <p>
+        To learn more about any of my previous projects please feel free to reach out to me
+    </p>
+</div></h2>
+<h2> Resume
+<div class="resume">
+    <ul class="resume">
+    <li class="resume">
```

Comment:
It's a little uncommon to see three different elements all with the same class. It would probably just work to give a class to the top-level element (the div in this case) and the children should inherit most of that (font size, color, etc.).

If you do need to style the specific tags underneath the div, you could do something like this:
```
.resume ul{
    /*anything you put here will be applied to any<ul> element that is a child of something with the resume class*/
}
```